packages/web/src/lib/web-mode.ts

```diff
@@ -149,6 +149,61 @@ export class WebMode {
 		return s;
 	}
 
+	/**
+	 * Append signature multisig transaction using algosigner
+	 * @param txns array of multisignature transaction with at least one signature
+	 * @param signers a subset of addresses to sign with the transaction
+	 * return an object contain blob attribute in base64
+	 */
+	async appendSignMultisigTransaction(txns: WalletTransaction[], signers: string[]): Promise<JsonPayload> {
+		const result:JsonPayload = {};
+		for (let i = 0; i < txns.length; ++i) {
```

Comment:
we could use `forEach` in here

Reply:
I do not use `forEach` here because I need to have the `i` variable for 
`result[i] = {
				blob: outputBase64
			};` in line 200